backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/barchart/BpkBarChartModel.kt

```diff
@@ -42,7 +41,8 @@ data class BpkBarChartModel(
    * @param title a primary text placed just below the bar itself.
    * @param subtitle a secondary text placed just below the title.
    * @param group name of the group in which the item exists. The name is rendered above the bars and updated as the chart scrolls horizontally.
-   * @param accessibilityLabel the label to be used for screen readers to read the values (e.g. "Monday 7, price is £100")
+   * @param contentDescription the content description to be used for screen readers to read the values (e.g. "Monday 7, price is £100")
```

Comment:
can you update the example to provide what needs to be included?

Reply:
to clarify: i meant the "Monday 7, price is £100" - also needs to include the month/group name here

Reply:
Done